contracts/teller-network/Escrow.sol

```diff
@@ -24,25 +26,44 @@ contract Escrow is Pausable {
         uint expirationTime;
         bool released;
         bool canceled;
+        bool paid;
         uint rating;
     }
 
     EscrowTransaction[] public transactions;
 
     License public license;
 
-    event Created(address indexed seller, address indexed buyer, uint escrowId);
+    address public arbitrer;
+
+    event Created(address indexed seller, address indexed buyer, uint escrowId, uint expirationTime);
     event Paid(uint escrowId);
+    event Released(uint escrowId);
     event Canceled(uint escrowId);
     event Rating(address indexed seller, address indexed buyer, uint escrowId, uint rating);
 
+    mapping(uint => ArbitrationCase) public arbitrationCases;
```

Comment:
Do you think Arbitration should be more like it's own contract
e.g Pausable, Arbitrable

Reply:
I'll review to see how can this be refactored, since setArbitrationResult depends on Escrow specific function. I guess some kind of delegation should be possible.